pkg/metrics/metrics.go

```diff
@@ -131,6 +131,22 @@ var (
 	// LabelKind is the kind a label
 	LabelKind = "kind"
 
+	// LabelPath is the label for the API path
+	LabelPath = "api_path"
+
+	// LabelAPIReturnCode is the HTTP code returned for that API path
+	LabelAPIReturnCode = "return_code"
+
+	// API interactions
+
+	// APIInteractions is the total time taken to process an API call made
+	// to the cilium-agent
+	APIInteractions = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "api_process_time",
```

Comment:
We name other histogram metrics like *_seconds, please do so here too.
```suggestion
		Name:      "api_process_time_seconds",
```